src/input.scss

```diff
@@ -9,17 +9,24 @@ $block: #{$fd-namespace}-input;
 
 .#{$block} {
   @include fd-form-text();
-  @include fd-var-size("height", $fd-forms-height, --fd-forms-height);
+  @include fd-var-size("height", $fd-forms-height, --sapElement_Height);
```

Comment:
once we consume the theming parameters, we don't have to call the mixin. The code would look:
```
height: var(--sapElement_Height);
line-height: var(--sapElement_Height);
```